src/Service/PostmarkService.php

```diff
@@ -188,6 +188,7 @@ public function send(Message $message): array
 
         $response = $this->prepareHttpClient('/email')
                          ->setMethod(HttpRequest::METHOD_POST)
+                         ->setEncType('application/json')
```

Comment:
Did you test this? To me application/json does not sound like an encoding type. This field I think should be reserved to set for example `multipart/form-data`.

To set application/json you would normally have to add a header with an accept type. So using the `setHeaders` method.

Reply:
Yes, it has been tested and has resolved our issues with Postmark. Postmark requires that the Content-Type be set to application/json.

<img width="695" alt="image" src="https://github.com/Webador/SlmMail/assets/205751/b6f3b0cd-bef2-4709-a0a5-9a03577f03de">


 I agree, it's not an encoding type per-se, it's a content type, but [Laminas HTTP uses this as the content type](https://github.com/laminas/laminas-http/blob/2.20.x/src/Client.php#L1248C31-L1248C31).

Reply:
Got it!